homeassistant/components/cloud/__init__.py

```diff
@@ -228,7 +228,8 @@ async def _on_connect():
 
     cloud.iot.register_on_connect(_on_connect)
 
-    await cloud.start()
+    hass.async_create_task(cloud.start())
```

Comment:
The reason it awaits, is that other integrations are using parts of the cloud as part of their setup. For example, requesting webhooks.

Reply:
This is not finished, not fully tested, and still very much a draft.
Main reason; Don't break everything when there are issues